### PR TITLE
Added Maximus of Tyre Dialexeis

### PR DESCRIPTION
**Pull request by sonofmun**

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |    +2310 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +66519 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0563/__cts__.xml`                          | New      |
| `./data/tlg0563/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0563/__cts__.xml`                          | Passing  |
| `./data/tlg0563/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3187)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `coverage`         |    -0.01 |
| `metadata_passing` |     5    |
| `metadata_total`   |     5    |
| `nodes_count`      |  2431    |
| `texts_passing`    |     2    |
| `texts_total`      |     3    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +72530 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0563/__cts__.xml`                          | New      |
| `./data/tlg0563/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1799/tlg015/__cts__.xml`                   | New      |
| `./data/tlg1799/tlg015/tlg1799.tlg015.opp-grc1.xml`   | New      |
| `./data/tlg2136/__cts__.xml`                          | New      |
| `./data/tlg2136/tlg001/__cts__.xml`                   | New      |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | New      |
| `./data/tlg0563/__cts__.xml`                          | Passing  |
| `./data/tlg0563/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | Passing  |
| `./data/tlg1799/tlg015/__cts__.xml`                   | Passing  |
| `./data/tlg1799/tlg015/tlg1799.tlg015.opp-grc1.xml`   | Failing  |
| `./data/tlg2136/__cts__.xml`                          | Passing  |
| `./data/tlg2136/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3398)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +5 |
| `metadata_total`   |       +5 |
| `nodes_count`      |    +2431 |
| `texts_passing`    |       +3 |
| `texts_total`      |       +3 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +72530 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0563/__cts__.xml`                          | New      |
| `./data/tlg0563/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1799/tlg015/__cts__.xml`                   | New      |
| `./data/tlg1799/tlg015/tlg1799.tlg015.opp-grc1.xml`   | New      |
| `./data/tlg2136/__cts__.xml`                          | New      |
| `./data/tlg2136/tlg001/__cts__.xml`                   | New      |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | New      |
| `./data/tlg0563/__cts__.xml`                          | Passing  |
| `./data/tlg0563/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | Passing  |
| `./data/tlg1799/tlg015/__cts__.xml`                   | Passing  |
| `./data/tlg1799/tlg015/tlg1799.tlg015.opp-grc1.xml`   | Passing  |
| `./data/tlg2136/__cts__.xml`                          | Passing  |
| `./data/tlg2136/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3416)*

**Comment by planatheisa** [comment]
There are two files for tlg1799.tlg015 in this PR. I don't think they're supposed to be merged with this.

@sonofmun

**Comment by sonofmun** [comment]
I had been working on them and accidentally added them to this PR. Go ahead and check them and, if they pass, go ahead and merge them.

File: data/tlg1799/tlg015/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg1799" xml:lang="grc" urn="urn:cts:greekLit:tlg1799.tlg015">
+  <ti:title xml:lang="lat">Sectio canonis [Sp.]</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg1799.tlg015.opp-grc1" workUrn="urn:cts:greekLit:tlg1799.tlg015">
+    <ti:label xml:lang="lat">Sectio canonis [Sp.]</ti:label>
+    <ti:description xml:lang="eng">Euclid, Sectio canonis [Sp.], Jahn, Teubner, 1895</ti:description>
```
Comment (planatheisa) [inline data/tlg1799/tlg015/__cts__.xml]: Should be "mul".

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +3 |
| `metadata_total`   |       +3 |
| `nodes_count`      |    +2330 |
| `texts_passing`    |       +2 |
| `texts_total`      |       +2 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +69063 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0563/__cts__.xml`                          | New      |
| `./data/tlg0563/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1799/tlg015/__cts__.xml`                   | New      |
| `./data/tlg1799/tlg015/tlg1799.tlg015.opp-grc1.xml`   | New      |
| `./data/tlg0563/__cts__.xml`                          | Passing  |
| `./data/tlg0563/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0563/tlg001/tlg0563.tlg001.1st1K-grc1.xml` | Passing  |
| `./data/tlg1799/tlg015/__cts__.xml`                   | Passing  |
| `./data/tlg1799/tlg015/tlg1799.tlg015.opp-grc1.xml`   | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3447)*

**Pull request merged**